app/models/hyrax/resource.rb

```diff
@@ -32,6 +32,7 @@ module Hyrax
   #
   class Resource < Valkyrie::Resource
     include Hyrax::Naming
+    include Hyrax::WithEvents
```

Comment:
How does this interact with the pull request? I'm not seeing specs surrounding changes to Hyrax::Resource getting "with_events"

Reply:
All of the event jobs rely on the ability to track and log those events for the affected resources. The Hyrax::WithEvents mixin adds that capability and is central to all of the specs for those jobs.

Reply:
The answer makes sense. My surprise came when I didn't see a change in spec to `hyrax/resource_spec.rb`

I'm wondering if there is a way to capture the expectation that Hyarx::Resource includes Hyrax::WithEvents. I know this relates to #4265 .

Reply:
I should add, that this is not a blocker for the PR. Just musing and wondering.

Reply:
Interesting. I somehow missed the whole conversation around 4265. I'll look into adding something that directly tests the new event requirement for Hyrax::Resource.